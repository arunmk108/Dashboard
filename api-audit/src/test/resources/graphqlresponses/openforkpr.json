{
    "data": {
        "repository": {
            "pullRequests": {
                "totalCount": 1,
                "pageInfo": {
                    "endCursor": "Y3Vyc29yOnYyOpK5MjAxNy0xMi0xMlQxMzoyNzozMC0wNTowMM4ADToh",
                    "hasNextPage": false
                },
                "edges": [
                    {
                        "cursor": "Y3Vyc29yOnYyOpK5MjAxNy0xMi0xMlQxMzoyNzozMC0wNTowMM4ADToh",
                        "node": {
                            "number": 1,
                            "mergeable": "MERGEABLE",
                            "state": "OPEN",
                            "createdAt": "2017-12-12T18:27:30Z",
                            "updatedAt": "2017-12-12T18:27:30Z",
                            "mergedAt": null,
                            "baseRef": {
                                "target": {
                                    "oid": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b"
                                }
                            },
                            "baseRefName": "master",
                            "title": "pr from fork",
                            "mergeCommit": null,
                            "headRepository": {
                                "name": "OpenForkPr",
                                "url": "https://mygithub.com/ghtwo/OpenForkPr"
                            },
                            "headRef": {
                                "target": {
                                    "oid": "5778be2d97000c3f5beb28cab25e9c905317f924"
                                }
                            },
                            "headRefName": "master",
                            "author": {
                                "login": "ghtwo",
                                "resourcePath": "/ghtwo"
                            },
                            "commits": {
                                "totalCount": 2,
                                "nodes": [
                                    {
                                        "commit": {
                                            "oid": "c91d42f07664b473a179f4bc7701fba6531511c6",
                                            "parents": {
                                                "nodes": [
                                                    {
                                                        "oid": "b2a0fd195a5322161e72cdcea66729d8b94da4f6"
                                                    }
                                                ]
                                            },
                                            "committedDate": "2017-12-12T18:26:52Z",
                                            "message": "Third fork commit",
                                            "status": null,
                                            "author": {
                                                "name": "Author, Two",
                                                "date": "2017-12-12T13:26:52-05:00",
                                                "user": {
                                                    "login": "ghtwo"
                                                }
                                            }
                                        }
                                    },
                                    {
                                        "commit": {
                                            "oid": "5778be2d97000c3f5beb28cab25e9c905317f924",
                                            "parents": {
                                                "nodes": [
                                                    {
                                                        "oid": "c91d42f07664b473a179f4bc7701fba6531511c6"
                                                    }
                                                ]
                                            },
                                            "committedDate": "2017-12-12T18:27:03Z",
                                            "message": "Fourth fork commit",
                                            "status": null,
                                            "author": {
                                                "name": "Author, Two",
                                                "date": "2017-12-12T13:27:03-05:00",
                                                "user": {
                                                    "login": "ghtwo"
                                                }
                                            }
                                        }
                                    }
                                ]
                            },
                            "comments": {
                                "totalCount": 0,
                                "nodes": []
                            },
                            "reviews": {
                                "totalCount": 0,
                                "nodes": []
                            }
                        }
                    }
                ]
            },
            "ref": {
                "target": {
                    "history": {
                        "pageInfo": {
                            "endCursor": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b 3",
                            "hasNextPage": false
                        },
                        "edges": [
                            {
                                "cursor": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b 0",
                                "node": {
                                    "oid": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "b0b3b4e5c8480a5c520e2864bb98f500573d76b8"
                                            }
                                        ]
                                    },
                                    "message": "Sixth direct commit",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2017-12-12T13:28:43-05:00"
                                    },
                                    "author": {
                                        "name": "Author, Two",
                                        "user": {
                                            "login": "ghtwo"
                                        },
                                        "email": "Author.Two@mycompany.com",
                                        "date": "2017-12-12T13:28:43-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b 1",
                                "node": {
                                    "oid": "b0b3b4e5c8480a5c520e2864bb98f500573d76b8",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "b2a0fd195a5322161e72cdcea66729d8b94da4f6"
                                            }
                                        ]
                                    },
                                    "message": "Fifth direct commit",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2017-12-12T13:28:30-05:00"
                                    },
                                    "author": {
                                        "name": "Author, Two",
                                        "user": {
                                            "login": "ghtwo"
                                        },
                                        "email": "Author.Two@mycompany.com",
                                        "date": "2017-12-12T13:28:30-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b 2",
                                "node": {
                                    "oid": "b2a0fd195a5322161e72cdcea66729d8b94da4f6",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "e3e1162e8abb353bcdf49f9d63f8f8f581df5b67"
                                            }
                                        ]
                                    },
                                    "message": "Second direct commit",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2017-12-12T13:26:08-05:00"
                                    },
                                    "author": {
                                        "name": "Author, Two",
                                        "user": {
                                            "login": "ghtwo"
                                        },
                                        "email": "Author.Two@mycompany.com",
                                        "date": "2017-12-12T13:26:08-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b 3",
                                "node": {
                                    "oid": "e3e1162e8abb353bcdf49f9d63f8f8f581df5b67",
                                    "parents": {
                                        "nodes": []
                                    },
                                    "message": "First direct commit",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2017-12-12T13:25:53-05:00"
                                    },
                                    "author": {
                                        "name": "Author, Two",
                                        "user": {
                                            "login": "ghtwo"
                                        },
                                        "email": "Author.Two@mycompany.com",
                                        "date": "2017-12-12T13:25:53-05:00"
                                    },
                                    "status": null
                                }
                            }
                        ]
                    }
                }
            },
            "issues": {
                "totalCount": 0,
                "pageInfo": {
                    "endCursor": null,
                    "hasNextPage": false
                },
                "edges": []
            }
        }
    }
}