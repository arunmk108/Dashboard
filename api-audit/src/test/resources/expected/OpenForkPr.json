[
    {
        "auditStatuses": [
            "SCM_AUTHOR_LOGIN_INVALID",
            "DIRECT_COMMITS_TO_BASE_FIRST_COMMIT",
            "DIRECT_COMMITS_TO_BASE"
        ],
        "lastUpdated": 1516207281058,
        "commits": [
            {
                "scmUrl": "https://mygithub.com/Devopscode/OpenForkPr",
                "scmBranch": "master",
                "scmRevisionNumber": "d0a5c26c3d5cfdff3cf8f925be933e0e936c533b",
                "scmCommitLog": "Sixth direct commit",
                "scmAuthor": "Author, Two",
                "scmAuthorLogin": "ghtwo",
                "scmParentRevisionNumbers": [
                    "b0b3b4e5c8480a5c520e2864bb98f500573d76b8"
                ],
                "scmCommitTimestamp": 1513103323000,
                "numberOfChanges": 1,
                "type": "New",
                "id": "5a5f77a16fa3ba5285b97547",
                "collectorItemId": "5a4e54f633ec296b81c3939a",
                "timestamp": 1516205985739,
                "firstEverCommit": false
            },
            {
                "scmUrl": "https://mygithub.com/Devopscode/OpenForkPr",
                "scmBranch": "master",
                "scmRevisionNumber": "b0b3b4e5c8480a5c520e2864bb98f500573d76b8",
                "scmCommitLog": "Fifth direct commit",
                "scmAuthor": "Author, Two",
                "scmAuthorLogin": "ghtwo",
                "scmParentRevisionNumbers": [
                    "b2a0fd195a5322161e72cdcea66729d8b94da4f6"
                ],
                "scmCommitTimestamp": 1513103310000,
                "numberOfChanges": 1,
                "type": "New",
                "id": "5a5f77a16fa3ba5285b97548",
                "collectorItemId": "5a4e54f633ec296b81c3939a",
                "timestamp": 1516205985739,
                "firstEverCommit": false
            },
            {
                "scmUrl": "https://mygithub.com/Devopscode/OpenForkPr",
                "scmBranch": "master",
                "scmRevisionNumber": "b2a0fd195a5322161e72cdcea66729d8b94da4f6",
                "scmCommitLog": "Second direct commit",
                "scmAuthor": "Author, Two",
                "scmAuthorLogin": "ghtwo",
                "scmParentRevisionNumbers": [
                    "e3e1162e8abb353bcdf49f9d63f8f8f581df5b67"
                ],
                "scmCommitTimestamp": 1513103168000,
                "numberOfChanges": 1,
                "type": "New",
                "id": "5a5f77a16fa3ba5285b97549",
                "collectorItemId": "5a4e54f633ec296b81c3939a",
                "timestamp": 1516205985739,
                "firstEverCommit": false
            },
            {
                "scmUrl": "https://mygithub.com/Devopscode/OpenForkPr",
                "scmBranch": "master",
                "scmRevisionNumber": "e3e1162e8abb353bcdf49f9d63f8f8f581df5b67",
                "scmCommitLog": "First direct commit",
                "scmAuthor": "Author, Two",
                "scmAuthorLogin": "ghtwo",
                "scmParentRevisionNumbers": [],
                "scmCommitTimestamp": 1513103153000,
                "numberOfChanges": 1,
                "type": "New",
                "id": "5a5f77a16fa3ba5285b9754a",
                "collectorItemId": "5a4e54f633ec296b81c3939a",
                "timestamp": 1516205985739,
                "firstEverCommit": true
            }
        ],
        "scmUrl": "https://mygithub.com/Devopscode/OpenForkPr",
        "scmBranch": "master"
    }
]